yarn-project/archiver/src/archiver/archiver_store.test.ts

```diff
@@ -64,10 +64,18 @@ describe('Archiver Memory Store', () => {
     const blocks = Array(10)
       .fill(0)
       .map((_, index) => L2Block.random(index));
-    await archiverStore.addL2Blocks(blocks);
-    await expect(async () => await archiverStore.getL2Blocks(1, 0)).rejects.toThrow(
-      `Invalid block range from: 1, limit: 0`,
-    );
+    await archiverStore.addBlocks(blocks);
+    await expect(async () => await archiverStore.getBlocks(1, 0)).rejects.toThrow(`Invalid limit: 0`);
+  });
+
+  it('returns from the beginning when "from" < genesis block', async () => {
+    const blocks = Array(10)
+      .fill(0)
+      .map((_, index) => L2Block.random(index));
+    await archiverStore.addBlocks(blocks);
+    const retrievedBlocks = await archiverStore.getBlocks(-5, 1);
```

Comment:
nit (for small brains like me) - the 2nd param in `getBlocks()` is `limit` right? Can you make that a variable so:
```
const limit = 1
const retrievedBlocks = await archiverStore.getBlocks(-5, limit);
 expect(retrievedBlocks.length).toEqual(limit);
```

As the code reads currently, I first thought the second param was `to`

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2816/commits/7721b0d9f7cf9d207a22e5bfe3fc909e95619231